src/ui/components/atoms/label-date/LabelDate.tsx

```diff
@@ -0,0 +1,14 @@
+import './LabelDate.scss';
+import React from 'react';
+import type { IProps } from './types/IProps';
+
+function LabelDate({ date, overflow = false }: IProps) {
+  const month = date.toLocaleString('es-ES', { month: 'long' });
+  const dayString = date.getDate().toString();
+  const yearString = date.getFullYear().toString();
```

Comment:
create a separate function to transform the date don't forget these are the shape of the date 2024-06-19T18:44:18.845Z is using the standard  ISO 8601